### ISSUE TITLE
Save error

### ISSUE THREAD

**Issue by monbvol** [MegaMek/mekhq#1708]
### Environment
Windows 10 Pro 64 bit
Java 1.8.0_221

### Description
When I tried saving my campaign MekHQ gave me an error about it not being able to save correctly.

Not sure if this is related to random procreation or not as I was able to save at previous points of the campaign and I know there is a bit of weirdness going on with the random marriage settings as I have it set to off for random same sex marriages but still wound up getting a couple during the campaign.

### Files
[megameklog.txt](https://github.com/MegaMek/mekhq/files/4560905/megameklog.txt)
[Enter the Fray.zip](https://github.com/MegaMek/mekhq/files/4560906/Enter.the.Fray.zip)
[Mine.zip](https://github.com/MegaMek/mekhq/files/4560918/Mine.zip)

**Comment by Windchild292** [comment]
Here is the fixed save, it was the random procreation issue. I have disabled random procreation for now, and would recommend leaving it off until 0.47.6.
[Enter the Fray.zip](https://github.com/MegaMek/mekhq/files/4561173/Enter.the.Fray.zip)

The random marriage bug (I flipped two operators accidentally) and save issue have both been fixed for the next release.

I apologise for the inconvenience.
